.github/workflows/Invoke-Pester.ps1

```diff
@@ -1,4 +1,5 @@
 Install-Module Pester,Microsoft.Powershell.SecretManagement -Force -Scope CurrentUser
+Set-Location -Path "$($PSScriptRoot)/../../SecretManagement.Keepass.Extension/Tests"
```

Comment:
Please remove this, you should do this work in your tests, this broke my tests by doing it.

Reply:
will do, thanks